app/data/models/backlog.js

```diff
@@ -22,11 +22,17 @@ module.exports = (sequelize, DataTypes) => {
     warnings: {
       type: DataTypes.TEXT,
       allowNull: true
+    },
```

Comment:
We can ignore this table as no longer used. In fact, I need to remove it now the access db import stuff has gone